pkg/drivers/metal/metal.go

```diff
@@ -124,6 +124,11 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 			Usage:  "The Equinix Metal Instance Termination Time",
 			EnvVar: "METAL_TERMINATION_TIME",
 		},
+		mcnflag.StringFlag{
+			Name:   "metal-ttl",
+			Usage:  "Set the Instance maximum Time To Live",
+			EnvVar: "METAL_TTL",
```

Comment:
I think `metal-ttl` could be misunderstood to be a `metal-api-request-ttl` or similar.

What do you think about extending the existing `metal-termination-time` field with https://github.com/carmo-evan/strtotime? I think this would enable the functionality you are looking for (and then some) while continuing to offer the existing date parsing behavior (which, as you've pointed out, never worked).

Reply:
Makes sense to have `metal-termination-time` support either a fixed time or duration.

I think `time.Duration` is convenient enough for most duration `1h`, `1d`, `90d`, `strtotime` can be convenient for a more flexible date format, especially when used in the command line if you don't mind the additional dependency.

Reply:
`time.Duration` only handles hours and smaller. https://play.golang.org/p/RouRQvFsLka

The dependency is thin (one visual page scan (lgtm), and no external dependencies).  The licenses (BSD-3 and MIT) are amicable.

Reply:
@jmarhee thoughts?

Reply:
@Miouge1 let's reuse `metal-termination-time` and change the parser out. with the strtotime parser your "1h","90d" snippets would need to be "1 hour", "90 day", etc, but we also get support for the existing format and added parsing abilities: "next week", "tomorrow", etc.

If you prefer, I can make these changes to your branch.

Reply:
Sounds good. I've updated the PR code and title to reflect the changes.

I have tested this with:
- `--metal-termination-time="10 minutes"`
- `--metal-termination-time="next week"`
- `--metal-termination-time="yesterday"` returns an error as expected: `Error setting machine configuration from flags provided: --metal-termination-time cannot be in the past`
- `--metal-termination-time="2006-01-02T15:04:05.000Z"` returns an error as expected: `Error setting machine configuration from flags provided: --metal-termination-time cannot be in the past`
- `--metal-termination-time="foobar"` returns an error as expected: `Error setting machine configuration from flags provided: strtotime: Unrecognizable input: "foobar"`